### DISCUSSION TITLE
Commercial Support for MS Build of OpenJDK

### DISCUSSION THREAD

**Discussion by wanchoonkit** [microsoft/openjdk#49] [Q&A] [answered]
I'd like to understand the Commercial Support for MS Build of OpenJDK.

**Comment by karianna** [comment, marked as answer]
@wanchoonkit - You can find our full support policy here: https://docs.microsoft.com/en-gb/java/openjdk/support - let us know if you have any further questions!

**Reply by wanchoonkit** [reply]
@karianna - Can I assume the commercial support is tagged along with Windows License? I'm not able to find any link.

**Reply by karianna** [reply]
Commercial support is for qualifying Azure and Azure Stack subscriptions.  There is currently no on-premise commercial support for Windows.  We are gathering more customer feedback on that use case, so please do feel free to email us at openjdk-support AT Microsoft DOT com

**Reply by KalleOlaviNiemitalo** [reply]
I hope this will be made clear in the [support roadmap](https://docs.microsoft.com/en-gb/java/openjdk/support), which currently does not mention Azure at all.

**Reply by karianna** [reply]
@brunoborges ^^ I understand we're updating this documentation set in the coming weeks which should add this clarification.
